index.html

```diff
@@ -1,413 +1,275 @@
+  <link rel="stylesheet" href="styles.css" />
+  <title>NEW DEVS ON THE BLOCK</title>
+</head>
+
+<body>
+  <header id="top">
+    <nav>
+      <a href="#about">About</a>
+      <a href="#schedule">Schedule</a>
+      <a href="#workshops">Workshops</a>
+      <a href="#coaches">Coaches</a>
+    </nav>
+    <img src="images/logo.png" class="logo" alt="kickass new devs on the block logo" />
+  </header>
+  <main>
+    <article id="about">
+      <h2>About us</h2>
+      <p>
+        Have you always wanted to build your first own website, but somehow, you didn't know how to get started? Maybe
+        because you thought you were not that techie after all? Then these two days are precisely for you! We will take the
+        first few steps down the web development road together.
+      </p>
+      <p>
+        NEW DEVS ON THE BLOCK is a two-day workshop for front-end development made and supported by the beautiful people
+        of the web community.
+      </p>
+      <p>
+        We will help you learn the fundamental building blocks of websites and guide you through coding your first
+        website yourself.
+      </p>
+    </article>
+
+    <article id="schedule">
+      <h2>Schedule</h2>
+      <p>
+        The workshop is divided into different topics that build on top of each other.
+        Each topic contains a theoretical part, where we go over some slides, and a coding part, where you will turn
+        theory into code.
+      </p>
+
+      <h3>Topics</h3>
+      <p>You will learn the following things</p>
+      <ul>
+        <li>History of the Web</li>
+        <li>Create the basic structure of a website</li>
+        <li>Add styling (i.e. fonts and colors) to your website</li>
+        <li>How to add images to your website</li>
+        <li>How to create layouts</li>
+        <li>How to add minimal animations</li>
+        <li>Use JavaScript to create a image gallery</li>
+      </ul>
+
+      <h3>Technical Topics</h3>
+      <p>We will cover the following technical aspects throughout the workshop</p>
+      <h4>HTML</h4>
+      <ul>
+        <li>Semantic HTML & Accessibility</li>
+        <li>Headings and document structure</li>
+        <li>Sectional elements</li>
+        <li>Text elements</li>
+      </ul>
+
+      <h4>CSS</h4>
+      <ul>
+        <li>Selectors</li>
+        <li>Pseudo-classes and elements</li>
+        <li>Colors</li>
+        <li>Flexbox</li>
+        <li>Transitions</li>
+      </ul>
+
+      <h4>JavaScript</h4>
+      <ul>
+        <li>Types</li>
+        <li>Functions</li>
+        <li>DOM API</li>
+      </ul>
+    </article>
+
+    <article id="workshops">
+      <h2>Workshops</h2>
+      <p>
+        Our next workshop will be in Vienna on April 18th & 19th 2020. The workshop will start at 10am.
+      </p>
+      <p>
+        If you wish to attend our workshop, <a href="https://forms.gle/kGtsSiLookczkHRq9">please fill out our
```

Comment:
Thanks for setting this up! 
I had a question about the location of the registration link. I was initially wondering if it might be hard to find, but then found that it's a good incentive to read the page's contents in detail.

Should it perhaps be a big button, though?

Reply:
Good point tho.

You are absolutely right, I guess the existing link is fine, but we should add something more catchy.

Reply:
My creativity budget is kind of empty right now.

I changed the label of the anchor tag to "Workshops & Register" - which is at least something 😄